docs/tableOfContents.js

```diff
@@ -81,36 +74,56 @@ export default [
         path: '/icons',
         title: 'Icons',
       }),
+      new Page({
+        path: '/inclusive',
+        title: 'Inclusive design',
+      }),
       new Page({
         path: '/layout',
-        title: 'Layout and grids',
-        disabled: true,
+        title: 'Layout',
       }),
       new Page({
-        path: '/forms',
-        title: 'Forms',
-        disabled: true,
+        path: '/writing',
+        title: 'Writing',
       }),
+    ],
+  }),
+  new Section({
+    title: 'Components',
+    autoSort: true,
+    pages: [
       new Page({
         path: '/buttons',
         title: 'Buttons and links',
       }),
-      new Page({
-        path: '/modals',
-        title: 'Modals',
-      }),
       new Page({
         path: '/loaders',
         title: 'Loaders',
       }),
+      new Page({
+        path: '/modals',
+        title: 'Modals',
+      }),
       new Page({
         path: '/snackbars',
         title: 'Snackbars',
       }),
     ],
   }),
+  new Section({
+    title: 'Patterns',
+    autoSort: true,
+    pages: [
+      new Page({
+        path: '/forms',
+        title: 'Forms',
+        disabled: true,
+      }),
+    ],
+  }),
```

Comment:
maybe 'Patterns' should be above 'Components'?

Then the ordering of sections would go from higher to lower levels of abstraction

Reply:
Done, makes sense